doc/api/http.md

```diff
@@ -766,11 +766,11 @@ added: v0.1.29
 * `callback` {Function}
 * Returns: {boolean}
 
-Sends a chunk of the body. By calling this method
-many times, a request body can be sent to a
-server — in that case it is suggested to use the
-`['Transfer-Encoding', 'chunked']` header line when
-creating the request.
+This method can be called multiple times. If no `Content-Length` is set, data
+will automatically be encoded in HTTP Chunked transfer encoding, so that
+server knows when data ends. The `Transfer-Encoding: chunked` header will be
+added. Call `end()` when finish sending data so that node would send the
+trailer.
```

Comment:
> If no `Content-Length` is set, data will automatically be encoded in HTTP 
Chunked transfer encoding, so that server knows when data ends. The `Transfer-Encoding: chunked` header will be automatically added.

Would be simpler maybe?

Reply:
With a reference to https://nodejs.org/api/stream.html#stream_writable_write_chunk_encoding_callback? Even though it is not a `Writable` the intention is to act as similar as possible. @mcollina, what do you think? Would such a reference be harmful?